devtools/cli/src/main/resources/templates/buildReport.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Quarkus Build Report - {buildTarget}</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
```

Comment:
I'm not sure about using an external CDN but given it's a dev feature, I'll let it slide and we will see if someone complains.

Reply:
Ah, yes, I completely forgot about this! I guess that I can remove this CSS...

Reply:
Yeah, if we can get a nice output with a couple of embedded CSS rules, that would have my preference.

Reply:
I'm complaining

Reply:
Done.